examples/pytorch_lightning_simple.py

```diff
@@ -81,58 +68,80 @@ def forward(self, data):
 class LightningNet(pl.LightningModule):
     def __init__(self, trial):
         super(LightningNet, self).__init__()
-        self.model = Net(trial)
 
-    def forward(self, data):
+        # We optimize the number of layers, hidden units in each layer and dropouts.
+        n_layers = trial.suggest_int("n_layers", 1, 3)
+        dropout = trial.suggest_float("dropout", 0.2, 0.5)
+        output_dims = [
+            trial.suggest_int("n_units_l{}".format(i), 4, 128, log=True) for i in range(n_layers)
+        ]
+
+        self.model = Net(dropout, output_dims)
```

Comment:
> Are you proposing to build a model inside Lightning?
> (Quote from the thread of the previous pull request of https://github.com/optuna/optuna/pull/2326#discussion_r572037238)

No, I'm not and sorry for putting my comment incongruously. I just thought it'd be more concise to use `torch.nn.Sequential` likewise https://github.com/optuna/optuna/blob/master/examples/kubernetes/mlflow/pytorch_lightning_distributed.py#L34  instead of defining custom `torch.nn.Module`. However, this is completely optional.

Reply:
I've just updated `class Net` implementation, thanks! 

About distributed example:
I' haven't seen this lol. Should I simplify this as well?

Reply:
I thought it seemed like a really good idea to save the logs. I'll make a commit now.